src/components/show-date/index.style.js

```diff
@@ -1,5 +1,3 @@
 import styled from 'styled-components';
 
-export const DateContainer = styled.div`
-  border: 1px solid red;
-`;
+export const DateContainer = styled.div``;
```

Comment:
we can omit this line and use a simple div in show-date/index.js file, so we can remove unnecessary import and export and make show-date/index.js file more simple